packages/@react-spectrum/menu/src/MenuTrigger.tsx

```diff
@@ -67,14 +67,43 @@ export function MenuTrigger(props: SpectrumMenuTriggerProps) {
     isOpen
   });
 
+  let isMobile = useMediaQuery('(max-width: 700px)');
```

Comment:
should we hook up anything to screen resizing? probably not since it'd be unlikely that someone would change the screen size while a menu is open, and i think it closes when you try something like that?

Reply:
`useMediaQuery` does listen for `matchMedia` events, so that should already happen. 😄